parameters/src/mainnet/mod.rs

```diff
@@ -41,7 +41,8 @@ impl_remote!(Degree24, REMOTE_URL, "resources/", "powers-of-beta-24", "usrs");
 impl_remote!(Degree25, REMOTE_URL, "resources/", "powers-of-beta-25", "usrs");
 impl_remote!(Degree26, REMOTE_URL, "resources/", "powers-of-beta-26", "usrs");
 impl_remote!(Degree27, REMOTE_URL, "resources/", "powers-of-beta-27", "usrs");
-impl_remote!(Degree28, REMOTE_URL, "resources/", "powers-of-beta-28", "usrs");
+// TODO (nkls): restore on CI.
+// impl_remote!(Degree28, REMOTE_URL, "resources/", "powers-of-beta-28", "usrs");
```

Comment:
Can you explain the PR why disabling these powers are safe? Can you show that they are not being used?

Reply:
Is this due to the limits on the size of the constraint system? 
If so, it would be nice to have a TODO wherever we specify the size limit to restore this when it is increased.
It's worth noting that we may eventually have to support these in CI.

Reply:
For reference @niklaslong the size limits are `MAX_DEPLOYMENT_CONSTRAINTS` and `MAX_VARIABLE_CONSTRAINTS`.